docs/components/modals/index.html

```diff
@@ -2,11 +2,9 @@
 title: Modals
 minver: 0.2.1
 also:
+  components/modals/test.html: Testing - Modals
   elements/hx-disclosure: <hx-disclosure>
   elements/hx-modal: <hx-modal>
```

Comment:
Add links to:
* Box Component ("Scrolling Containers") (`components/box/#scrolling-containers`)
* `<hx-div>` (`elements/hx-div`)